paths/electron/starting-with-electron/02-electron-boilerplate.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Electron Boilerplate
+permalink: /electron/starting-with-electron/boilerplate
+next-page: /electron/starting-with-electron/init-git-repository
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  Right now, our directory has no files. We want it to be set up as a base electron boilerplate called [electron-forge](https://electronforge.io/). To do that, we need to go through a few steps.
+
+  Important files to watch out for:
+  - package.json
+  - src/index.html
+  - src/index.json
+
```

Comment:
We may also want to surround them in single backticks for consistency with `04-creating-app-files.md`.